app/serializers/basic_theme_serializer.rb

```diff
@@ -10,4 +10,8 @@ def include_default?
   def default
     true
   end
+
+  def description
```

Comment:
Seems fine to move this into the basic version, I would consider the description fairly basic